### PR TITLE
DBZ-8013 Add async. engine blog post

### PR DESCRIPTION
**Pull request by vjuranek**
https://issues.redhat.com/browse/DBZ-8013

### CODE REVIEW

File: _posts/2024-07-08-async-embedded-engine.adoc [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+Later on, a public API for `DebeziumEngine` was defined ( https://issues.redhat.com/browse/DBZ-234[DBZ-234]).
+The interface decoupled the user-facing API from the implementation and provided the ability to replace the implementation by a different one.
+Shortly after introducing the Debezium engine API, a new Debezium server was created as part of https://issues.redhat.com/browse/DBZ-651[DBZ-651].
+While `EmbeddedEngine` had to be wrapped into another application which would consume the records, the Debezium server provided a way to run Debezium outside the Kafka Connect cluster without such a wrapper.
+The Debezium server provided several sinks, so the users could use the server out-of-the-box without any further coding, but the Debezium server is still powered by the Debezium engine.
+As the popularity of the Debezium server increased, more and more sinks were added.
+This resulted in the separating Debezium server into https://github.com/debezium/debezium-server/[separate GitHub project].
+The separation of the Debezium server repository happened relatively recently ( https://issues.redhat.com/browse/DBZ-6049[DBZ-6049]).
```
Comment (Naros) [inline _posts/2024-07-08-async-embedded-engine.adoc]: I wonder as a part of this history lesson, would it make sense to also include the evolution of the Operator here just as a tidbit to let users be aware that Debezium Server is the runtime under the K8 operator too?  If so, then I think this gives users 3 options, where two slightly differ only between standalone vs K8s.

**Review by Naros — COMMENTED** [review]
Just one tiny comment, overall LGTM.

File: _posts/2024-07-08-async-embedded-engine.adoc (hunk 1 above)
Comment (vjuranek) [reply _posts/2024-07-08-async-embedded-engine.adoc]: @Naros thanks, good idea! Will add it.

Comment (vjuranek) [reply _posts/2024-07-08-async-embedded-engine.adoc]: done

**Comment by mfvitale** [comment]
Good one.

**Review by Naros — APPROVED** [review]
LGTM @vjuranek, feel free to merge it yourself if its good from your PoV.

**Comment by vjuranek** [comment]
Merged, thanks for the reviews @Naros and @mfvitale !

**Pull request merged**